src/server/api/routes/users/v2/notification.js

```diff
@@ -0,0 +1,65 @@
+const { Router } = require('express');
+const {
+  authenticateV2Login,
+  checkParamWalletMatch,
+  checkEmailHasVerified,
+} = require('../../../middleware/auth');
+const { setPrivateCacheHeader } = require('../../../middleware/cache');
+const { handleRestfulError } = require('../../../middleware/error');
+const { walletUserCollection } = require('../../../../modules/firebase');
+
+const router = Router();
+
+router.get(
+  '/:wallet/notification',
+  authenticateV2Login,
+  checkParamWalletMatch,
+  async (req, res, next) => {
+    try {
+      const { wallet: user } = req.params;
+      const userDoc = await walletUserCollection.doc(user).get();
+      const { notification = {} } = userDoc.data();
+      const { transfer = false } = notification;
+      res.json({
+        notification: {
+          transfer,
+        },
+      });
```

Comment:
Just return `notification` object? preferably with a filter to filter out secret (if we do have)

Reply:
Do we need to gerantee the fields in `notification` for this API? Or just let front-end decide the action if it's `undefined`?

Reply:
yeah having defaults here could be useful, lets keep it then